catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -665,11 +663,24 @@ protected ResourceInfo getResourceInfo(ResourceRequest resourceRequest, String s
     }
 
     private Query anyTag(Query query) {
+        //Any metacard tag
         Filter anyTag = frameworkProperties.getFilterBuilder()
                 .attribute(Metacard.TAGS)
                 .is()
                 .like()
                 .text(FilterDelegate.WILDCARD_CHAR);
+
+        //no metacard tag
+        Filter nullTag = frameworkProperties.getFilterBuilder()
+                .attribute(Metacard.TAGS)
+                .is()
+                .empty();
+
+        //any or no metacard tag
+        anyTag = frameworkProperties.getFilterBuilder()
+                .anyOf(anyTag, nullTag);
```

Comment:
Just a bit, don't overwrite the variable. Makes it harder to follow. Declare a new var for final filter.